common/lib/xmodule/xmodule/js/src/video/02_html5_video.js

```diff
@@ -346,8 +313,6 @@ function(_) {
 
             // Place the <video> element on the page.
             this.videoEl.appendTo(el.find('.video-player > div:first-child'));
-
-            return true;
```

Comment:
This should be removed. I added this when i reverted the cleanup changes earlier.